### ISSUE TITLE
[saas-11.5] Localization Engine Issue

### ISSUE THREAD

**Issue by jlzhou** [odoo/odoo#27314]
Impacted versions:
11.5 (850e1675)

Steps to reproduce:
Initialize a new odoo db without installing any app modules.
Load a translation such as zh_CN (Chinese Simplified) in Settings.
Set User Language as above new loaded one in Preferences.
Click Menu "Settings - User & Companies - Users" (Actually the menu name shows in localized language, such as “设置 - 用户 & 公司 - 用户”)

Current behavior:
odoo reports Server Error as below,
...
ValueError: 字段`sel_groups_1_9_10`不存在

错误的上下文:
视图 `res.users.form`
[view_id: 144, xml_id: base.view_users_form, model: res.users, parent_id: n/a]

That means: ValueError: Field `sel_groups_1_9_10` does not existed

Expected behavior:
Show the User tree view as normal.

Video/Screenshot link (optional):
![1538268526142](https://user-images.githubusercontent.com/1243299/46251719-ae63a980-c48d-11e8-8862-e25b354ffa2e.jpg)

**Comment by jlzhou** [comment]
Tip: A way to walk around it:
Remove the translation of "Internal User" and "Portal" in base module.

**Comment by niulinlnc** [comment]
This problem does not only appear in the Chinese language pack, all other non-English language packs, the same problem will occur if these two terms are translated.
![1](https://user-images.githubusercontent.com/11767049/46252020-df93a800-c494-11e8-9a18-f2449fe702e4.jpg)
![2](https://user-images.githubusercontent.com/11767049/46252021-e4f0f280-c494-11e8-8afd-33f34c2c085f.jpg)

**Comment by nhomar** [comment]
**solution**

Go to transifex and fix the strings that were incorrectly translated, I found at least 1 in spanish.

**Technical reason:** https://github.com/odoo/odoo/issues/27311

**Comment by jlzhou** [comment]
@nhomar I believe this is not a translation bug, and it is NOT a duplicate issue, please see my report carefully.

**Comment by nhomar** [comment]
@jlzhou 

Yes my friend, it is the same root, there is an xml that weent into the trasnlation platform and in the activation process of v12 it cleans up several little views left them broken.

Is itt working on english? if YES then it IS a translation issue, BUT it is not a problem in general for everybody (the translators can fix it) but for the translation engine which should trigger a secure test and not avoid the load of a view, on my issue the error was in the accounting dashboar, but it is the same root.

I hope it clarifies my clasification, if you are not agree I am here to review it again.

Rergards.

**Comment by jlzhou** [comment]
@nhomar 

Thanks for your reply,.

The issue does like other translation issue, we can not add any translation on these terms:
"Internal User"(model:res.groups,name:base.group_user) 
"Portal"(model:res.groups,name:base.group_portal)
but they surely need to be translated.

Please do replay my steps, you will know what I mean.

Regards

**Comment by jlzhou** [comment]
@mart-e This is a critical issue that has been existing for a long time.

**Comment by niulinlnc** [comment]
I agree with zhou, for example, the field name "public" of the third option can be translated without any error!
